When scanning large resource-group - creates PANIC and crashes

When trying to scan/import a resource group, as it's listing the resources after about 15-20s will give the error:

```
$ aztfy rg myResourceGroup
  Azure Terrafy
  ⡿  Initializing...
  ⡿  Listing Azure Resources...
panic: interface conversion: interface {} is nil, not string

goroutine 181 [running]:
github.com/magodo/azlist/azlist.listDirectChildResource({0x10441c210, 0x14000124010}, 0x14001f10050, 0x100000000000000?, {{0x104420718?, 0x1400004c140?}, 0x14002177c20?})
	github.com/magodo/azlist@v0.0.0-20220923015326-17b16cc9b805/azlist/azlist.go:372 +0x5b8
github.com/magodo/azlist/azlist.ListChildResource.func2()
	github.com/magodo/azlist@v0.0.0-20220923015326-17b16cc9b805/azlist/azlist.go:285 +0x3c
github.com/magodo/workerpool.(*workPool).Run.func1(0x0?)
	github.com/magodo/workerpool@v0.0.0-20211124060943-1c48f3e5a514/workerpool.go:75 +0x78
created by github.com/magodo/workerpool.(*workPool).Run
	github.com/magodo/workerpool@v0.0.0-20211124060943-1c48f3e5a514/workerpool.go:66 +0x50
```


```
$ az version
{
  "azure-cli": "2.41.0",
  "azure-cli-core": "2.41.0",
  "azure-cli-telemetry": "1.0.8",
  "extensions": {}
}

$ brew list aztfy
/opt/homebrew/Cellar/aztfy/0.8.0/bin/aztfy

$ aztfy -version
aztfy version v0.8.0(a7c179f)

$ go version
go version go1.19 darwin/arm64

$ terraform version
Terraform v1.2.7
on darwin_arm64
```

--- a/go.mod
+++ b/go.mod
@@ -16,7 +16,7 @@ require (
 	github.com/hashicorp/terraform-exec v0.17.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/magodo/armid v0.0.0-20220923023118-aec41eaf7370
-	github.com/magodo/azlist v0.0.0-20221012021422-efffc8a52108
+	github.com/magodo/azlist v0.0.0-20221028055654-026a60de1bc9
 	github.com/magodo/aztft v0.3.1-0.20221024073545-bf35b3f23e5e
 	github.com/magodo/spinner v0.0.0-20220720073946-50f31b2dc5a6
 	github.com/magodo/textinput v0.0.0-20210913072708-7d24f2b4b0c0

